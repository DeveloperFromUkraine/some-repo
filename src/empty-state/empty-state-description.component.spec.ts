import { ComponentFixture, TestBed } from "@angular/core/testing";
import { EmptyStateDescriptionComponent } from "index";
import { ComponentTest } from "../../test/test-bed/component";

describe('EmptyStateDescriptionComponent', () => {
    let fixture: ComponentFixture<EmptyStateDescriptionComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [EmptyStateDescriptionComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(EmptyStateDescriptionComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});