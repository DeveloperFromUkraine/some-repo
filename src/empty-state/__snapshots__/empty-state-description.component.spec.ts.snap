// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyStateDescriptionComponent should match snapshot 1`] = `
<ign-empty-state-description>
  <div
    _ngcontent-c0=""
  />
</ign-empty-state-description>
`;
